bin/check-vault-status.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+#
+# check-vault-status
+#
+# DESCRIPTION:
+#   This plugins checks if vault is up & reachable. It then checks
+#   if the vault is unsealed
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: vault
+#
+require 'sensu-plugin/check/cli'
+require 'vault'
+
+class VaultStatus < Sensu::Plugin::Check::CLI
+  option :host,
+         description: 'vault host',
+         short: '-h HOST',
+         long:  '--host HOST',
+         default: '127.0.0.1'
+
+  option :port,
+         description: 'vault port',
+         short: '-p PORT',
+         long: '--port PORT',
+         default: '8200'
+
+  option :scheme,
+         description: 'vault http scheme',
+         short: '-s SCHEME',
+         long: '--scheme SCHEME',
+         default: 'https'
+
```

Comment:
Since we know what values are supported we should restrict them so if someone typos the error they get a more helpful error:
```suggestion
         default: 'https',
         in: %w(https http)
```